### PR TITLE
Change Language

### PR DESCRIPTION
**Pull request by Guiorgy**
- Introduced 2 new strings: change_language & system_default
- I reused LanguageActivity, but the one in General tab changes the app locale and also includes System Default option
- Added a LocaleManager, and overloaded attachBaseContext & onConfigurationChanged
- __NOTE!__ also overloaded applyOverrideConfiguration in BaseActivity , __BUT__ this is a workaround for a bug in appcompat 1.1.0 and should be removed once it has been updated to 1.2.0, or downgraded to 1.0.2

Also, I noticed you were calling MultiDex.install(this), even though it is called in super anyways. I removed it, but if I shouldn't have, do tell me

### CODE REVIEW

**Comment by Guiorgy** [comment]
Works great on my Amazon Fire, but now that I tested it on my samsung something's wrong...

**Comment by Guiorgy** [comment]
So far works fine. I'll be running the app with #81 and #83 for the time being, hope there are no issues

File: app/src/main/java/io/github/gmathi/novellibrary/NovelLibraryApplication.kt [hunk 1]
```diff
@@ -159,9 +159,12 @@ class NovelLibraryApplication : MultiDexApplication() {
     }
 
     override fun attachBaseContext(base: Context) {
-        super.attachBaseContext(base)
-        MultiDex.install(this)
```
Comment (gmathi) [inline app/src/main/java/io/github/gmathi/novellibrary/NovelLibraryApplication.kt]: We don't need this? `MultiDex.install(this)`

Comment (Guiorgy) [reply app/src/main/java/io/github/gmathi/novellibrary/NovelLibraryApplication.kt]: this is what super.attachBaseContext looks like:
``` java
public class MultiDexApplication extends Application {
  @Override
  protected void attachBaseContext(Context base) {
    super.attachBaseContext(base);
    MultiDex.install(this);
  }
}
```
Personally, I have never used Multidex, just normal Application

**Comment by Guiorgy** [comment]
@gmathi, my Discord is Guiorgy#6962

File: app/src/main/java/io/github/gmathi/novellibrary/NovelLibraryApplication.kt (hunk 1 above)
Comment (Guiorgy) [reply app/src/main/java/io/github/gmathi/novellibrary/NovelLibraryApplication.kt]: > If you do override the Application class, change it to extend MultiDexApplication (if possible) as follows:
```kotlin
class MyApplication : MultiDexApplication() {...}
```
If you don't extend MultiDexApplication then:
```kotlin
class MyApplication : SomeOtherApplication() {
    override fun attachBaseContext(base: Context) {
        super.attachBaseContext(base)
        MultiDex.install(this)
    }
}
```
so no need for MultiDex.install(this)

Comment (gmathi) [reply app/src/main/java/io/github/gmathi/novellibrary/NovelLibraryApplication.kt]: Ah! missed that detail!

**Pull request merged**